pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -1102,6 +1102,21 @@ self: super: builtins.intersectAttrs super {
     broken = false;
   }) super.cabal-install;
 
+  tailwind = addBuildDepend
+      # Overrides for tailwindcss copied from:
+      # https://github.com/EmaApps/emanote/blob/master/nix/tailwind.nix
+      (pkgs.nodePackages.tailwindcss.overrideAttrs (oa: {
+        plugins = [
+          pkgs.nodePackages."@tailwindcss/aspect-ratio"
+          pkgs.nodePackages."@tailwindcss/forms"
+          pkgs.nodePackages."@tailwindcss/language-server"
```

Comment:
I think we can remove `"@tailwindcss/language-server"`. But the other plugins are generally used. Pretty much anything you can use in the Emanote HTML templates as a user can be added here.

Reply:
> Anyway, @srid I am a bet weary about the tailwindcss overrides. I have just copied them, but do you maybe have a bit of documentation for why those are needed. I looked for that in the tailwind README on hackage. I don‘t like the idea, that this is bespoke knowledge only present in your flake and nixpkgs.

I should probably expand the README of https://github.com/srid/tailwind-haskell .... but since `tailwind` Haskell package is responsible for mainly compiling the classes to produce a final `.css`, I would say depending on all plugins that are useful in that compilation step is what is relevant. That means (AFAIU) all the plugins here except `langauge-server`.